### ISSUE TITLE
Form validation

### ISSUE THREAD

**Issue by saadaouad** [erikras/redux-form#2236]
Hello,
I need to show an `message error` if `input name` is empty.
You will find below my react file and the error displayed in the console for chrome : 

Hire-us.js  
```bash
import React, { Component, PropTypes } from 'react';
import Helmet from 'react-helmet';
import { reduxForm } from 'redux-form';
import _ from 'lodash';

const FIELDS = {
  userName: {
    type: 'input',
    label: 'Name for person'
  }
}

class HireUs extends Component {
    onSubmit(props) {
      alert('Thanks');
    }
    render() {
        const{ fields: { userName }, handleSubmit } = this.props;

        return (
          <div className='hire-us container'>
            <Helmet title="Hire us to help you realize your idea"/>
            <main>
              <section className="hero clearfix">
                <h1>Hire us to help you realize your idea</h1>
                <p>Let's get in touch today, We're excited to hear from you!</p>
                <form onSubmit={handleSubmit(props => this.onSubmit(props))} >
                  <div className={'form-group'}>
                    <label>What is your name ?</label>
                    <input type="text" className="form-control" {...userName} />
                    { userName.touched && userName.error && <div> { userName.error } </div> }
                  </div>
                  <button type="submit">Submit</button>
                </form>
              </section>
            </main>
          </div>
        );
      }
}

function validate(values) {
  const errors = {};
  if (!values.userName) {
    errors.userName = 'Name is required';
  }
  return errors;
}

HireUs.PropTypes = {
  fields : PropTypes.object.isRequired,
  handleSubmit : PropTypes.func.isRequired
};

export default reduxForm({
  form: 'HireUs',
  fields: _.keys(FIELDS),
  validate
})(HireUs);
```

Error
```bash
Uncaught TypeError: Cannot read property 'touched' of undefined
Uncaught TypeError: Cannot read property '__reactInternalInstance$1yra2hb0l1kux74pqspxcx47vi' of null
```
You can help me please.

**Comment by dmeenhuis** [comment]
Judging by your example, it seems you forgot to wrap your input with ``Field``. So right now it's not connected to ReduxForm.

See here for details: http://redux-form.com/6.3.1/docs/api/Field.md/. Otherwise refer to the sync validation example here: http://redux-form.com/6.3.1/examples/syncValidation/.

**Comment by saadaouad** [comment]
Effectively, your right.
Thanks for your help Dennis :+1:

**Comment by erikras** [comment]
Published fix in [`v6.4.2`](https://github.com/erikras/redux-form/releases/tag/v6.4.2).
